app/views/broker_agencies/broker_roles/_new_broker_agency_form.html.erb

```diff
@@ -22,6 +22,11 @@
             <button type="submit" class="btn btn-primary pull-right create_broker_agency_btn interaction-click-control-create-broker-agency">Create Broker Agency</button>
           </div>
         </div>
+        <div class="row">
+          <div class="col-xs-12">
+            <h5>By submitting this application to become a certified broker on the Health Connector for Business, I agree to the terms and conditions described in the <%= site_brokers_agreement_path %>.</h5>
```

Comment:
@brianweiner they wanted this text to be above  create Broker Agency button and below phone.